tests/Jenkins/dirac_ci.sh

```diff
@@ -130,18 +133,23 @@ function installSite(){
     } >> "$SERVERINSTALLDIR/dirac-ci-install.cfg"
   fi
 
-  if [ "$ALTERNATIVE_MODULES" ]; then
+  if [ -n "${ALTERNATIVE_MODULES+x}" ]; then
     echo "Installing from non-release code"
-    if [[ -d "$ALTERNATIVE_MODULES" ]]; then
-      installOptions+="--module=$ALTERNATIVE_MODULES:::DIRAC:::local"
-    else
-      installOptions+="--module=$ALTERNATIVE_MODULES"
-    fi
+    option="--module="
+    for module_path in "${ALTERNATIVE_MODULES[@]}"; do
+      if [[ -d "${module_path}" ]]; then
+        option+="${module_path}:::$(basename "${module_path}"):::local,"
+      else
+        option+="${module_path},"
+      fi
+    done
+    INSTALLOPTIONS+=("${option: :$((${#option} - 1))}")
   fi
 
-  echo "==> Installing with options $installOptions $SERVERINSTALLDIR/install.cfg"
+  # HACK: dirac-install.py currently puts installation symlinks in the current working directory
+  cd "${SERVERINSTALLDIR}" || exit 1
```

Comment:
This is a workaround for https://github.com/DIRACGrid/DIRAC/pull/4430

Reply:
So, if #4430 is merged, will you rebase and remove this hack?
Now, in https://github.com/DIRACGrid/DIRAC/pull/4436#issuecomment-583028471 there's a comment about another bug fix that should be in this same PR, and for which @andresailer committed yet another "hack/temp fix" (https://github.com/DIRACGrid/DIRAC/pull/4436/commits/1c342128472cad2cd1d17e54ddc99bd9eceb1733) that will be removed after.
So, can you please add to #4430 the fix mentioned above? Then I will merge, propagate, and both of you can fix your respective PRs.

Reply:
Okay. I've applied the commit to #4430 and the CI has passed.